jekyll/_cci2/gh-bb-integration.md

```diff
@@ -86,24 +86,51 @@ ERROR: Repository not found.
 Permission denied (publickey).
 ```
 
+## Machine User Keys
+
+For security reasons,
+consider creating a machine user
+for your CircleCI projects.
+A [machine user](https://developer.github.com/v3/guides/managing-deploy-keys/#machine-users) is a GitHub user
+that you create for running automated tasks.
+By using the SSH key of a machine user,
+you allow anyone with repository access
+to build, test, and deploy the project
+without sacrificing security.
+To use the SSH key of a machine user,
+follow the steps below.
 
-<h2 id="machine-user-keys">Machine user keys</h2>
-
-Consider using a machine user's SSH key instead of a regular user's key for automated tasks that may have restricted access to required repos. A [machine user](https://developer.github.com/guides/managing-deploy-keys/#machine-users) is a GitHub user which you create only for automated tasks.  Add a machine user's SSH key to your projects on CircleCI and use that key as the *Checkout SSH key* for these projects, instead of using deploy keys or your own SSH keys.
+**Note:**
+To perform these steps,
+the machine user must have admin access.
+When you have finished adding projects,
+you can revert the machine user to read-only access.
 
-Here are the steps to set a machine user's SSH key as a checkout key for your project. **Note:** An account with admin privileges of the organization is required to perform these steps.
+1. Create a machine user
+by following the [instructions on GitHub](https://developer.github.com/v3/guides/managing-deploy-keys/#machine-users).
 
-1. Log in to GitHub as the machine user.
+2. Log in to GitHub as the machine user.
 
-2. Go to <https://circleci.com> and log in. GitHub will ask you to authorize CircleCI to access the machine user's account, so click on the **Authorize application** button.
+3. [Log in to CircleCI](https://circleci.com/login).
```

Comment:
this is where the use was confused, shouldn't we add here to log in to CCI as the machine user?

Reply:
Hmm... that's why I added the previous steps ("create a machine user", "log into GitHub as the machine user"). When you log into CircleCI and authorize with GitHub, it will use the GitHub account you're logged into. There are more steps this way, but less room for user error. Thoughts?